faw/docker-bin/faw-db-console.py

```diff
@@ -3,12 +3,18 @@
 import code
 import os
 import pymongo
+import readline
+import rlcompleter
 
 # Connect a pymongo instance with 'db' being the database corresponding to this
 # instance.
 _c = pymongo.MongoClient()
 db = _c[os.environ['DB']]
 
-print(f'Use the `db` variable to interact with the database')
-code.interact(local=dict(globals(), **locals()))
+# Enable tab completion in the REPL
+vars = {**globals(), **locals()}
+readline.set_completer(rlcompleter.Completer(vars).complete)
```

Comment:
I'm also a fan of ipython for things like this--my only concern would be that it could be overkill to add a somewhat heavy dependency when the stdlib is serviceable.

Swapping ipython in here would add much nicer handling of multiline statements, and the ability to `await` at top level, among other ipython-isms. The main costs would be heavier overhead (startup is slower than the stdlib repl) and dependencies--I've run into conflicts between ipython and core deps on my projects a few times in the past. This is unlikely to come up for the FAW though since we don't pin deps.

(Unimportant: Embedding into the surrounding namespace isn't necessary here, so I'd use `IPython.start_ipython` instead of `IPython.embed` in this case)